### ISSUE TITLE
Fixed in latest valve csgo patch

### ISSUE THREAD

**Issue by fishjojo1** [kkthxbye-code/csgo_bugs#7]
The latest valve game patch has fixed this issue.
"– sv_pure servers will now kick clients that have loaded asset files into game memory from files not present on game server filesystem."
https://blog.counter-strike.net/index.php/category/updates/ (check 5/4/2020)
sv_pure servers now compare in game memory with default(server) memory.
Not sure if replacing the file mid game will work. Doubt so as it seems to me that the server has timed checks. Have not tried though. 

TL;DR : you'll get kicked if you try to add the soundmixers.txt file in /platform , as the server checks it now.

**Comment by theFreeWall** [comment]
"not present on game server filesystem"
These files are present on game server filesystem.It ’s just that they are broken(by corrupting the checksum of a vtx)and different from server's.
We need to test.

**Comment by ogges** [comment]
its fixxed

**Comment by szmarczak** [comment]
https://github.com/kkthxbye-code/csgo_bugs/pull/6#issuecomment-624104555

**Comment by fishjojo1** [comment]
Confirmed not working, closing issue now.
